src/generate_container_host_image.py

```diff
@@ -111,6 +162,10 @@ def parse_args(zone, project):
 			)
 
 			print("Creating image from snapshot ...")
+			try:
+				subprocess.check_call("gcloud compute images delete --quiet {imagename}".format(imagename = imagename), shell = True)
+			except subprocess.CalledProcessError:
+				pass
```

Comment:
Is this something to worry about?

Reply:
I think if people run `setup.sh` twice, and if the previous image exists, it will fail. I don't know if we want to skip building image if old image exists or delete the old image (the latter made testing easier for me).